### PR TITLE
Fix definition for class

### PR DESCRIPTION
**Pull request by mpollmeier**
I'm new to scalameta, but this looks like a typo. The proposed version matches fine on my class, the previous results in a MatchError

### CODE REVIEW

**Comment by olafurpg** [comment]
Thanks for this PR! The primary constructor for a class can have modifiers

```scala
> val q"..$classMods class $tname[..$tparams] ..$constructorMods (...$paramss) extends $template" = q"class A private (a: Int)"
classMods: collection.immutable.Seq[Mod] = List()
tname: Type.Name = A
tparams: collection.immutable.Seq[Type.Param] = List()
constructorMods: collection.immutable.Seq[Mod] = List(private) // <-- here
paramss: collection.immutable.Seq[collection.immutable.Seq[Term.Param]] = List(List(a: Int))
template: Template =
```

I agree the docs should not define `mods` twice, however. Any idea what would be better name for the second `mods`?

**Comment by mpollmeier** [comment]
That makes sense, thanks!

So we should change the naming in the docs since it can't ever work right, since it's using the same name for two variables. I would not change the first `mods` since it's nicely aligned with the other examples in this section :)
How about we just rename the second one to `constructorMods` or `ctorMods`? I can update the PR - just signed the CLA as well.

**Comment by olafurpg** [comment]
I think `ctorMods` is more consistent with the scala.meta naming convention. LGTM 👍 thanks!

File: notes/quasiquotes.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Literal  | `q"$lit"` (construction only), `q"${lit: Lit}"` (also deconstruction)
  Def            | `q"..$mods def $name[..$tparams](...$paramss): $tpeopt = $expr"`
  Macro          | `q"..$mods def $name[..$tparams](...$paramss): $tpeopt = macro $expr"`
  Type           | `q"..$mods type $tname[..$tparams] = $tpe"`
- Class          | `q"..$mods class $tname[..$tparams] ..$mods (...$paramss) extends $template"`
+ Class          | `q"..$mods class $tname[..$tparams] $ctorMods (...$paramss) extends $template"`
```
Comment (xeno-by) [inline notes/quasiquotes.md]: Is the change from `..$` to `$` accidental or intentional?

Comment (xeno-by) [inline notes/quasiquotes.md]: /cc @olafurpg

**Pull request merged**